tests/test_statemanager.py

```diff
@@ -509,6 +527,24 @@ def test_transition_from_none(self):
         self.assertFalse(self.post.reviewstate.LOCKED)
         self.assertTrue(self.post.state.PENDING)
 
+    def test_transition_aborterror(self):
+        """transition method aborts intentionally"""
+        self.post.submit()
+        self.assertTrue(self.post.state.PENDING)
```

Comment:
Include this assertion before as well, so it's obvious to the reader the state hasn't changed.